Modix.Services.Test/MediatrIntegrationTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Threading.Tasks;
+using Discord;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+using Modix.Services.Core;
+using Modix.Services.Messages.Discord;
+using Modix.Services.Moderation;
+using Moq;
+using Moq.AutoMock;
+using NUnit.Framework;
+
+namespace Modix.Services.Test
+{
+    public class MediatrIntegrationTests
+    {
+        [Test]
+        public async Task MediatrCanResolveNotificationHandlers()
```

Comment:
Add a .editorconfig file to the test project to ignore the naming rule here (I.E. copy the one from `Modix.Data.Test`).

Reply:
Test naming should be (at least, it's the pattern I follow in Modix.Data.Test) `UnitUnderTest_TestConditions_ExpectedOutput`. So, for this, I would say it should be `Publish_Always_PublishesToNotificationHandlers` or similar. Then you can think about what test conditions might exist, other than "Always".